main.go

```diff
@@ -269,12 +329,65 @@ func _main() error {
 		return err
 	}
 	defer os.RemoveAll(tmpDir)
+	if cache {
+		tmpDir = "dsq-cache"
+	}
+
+	settings := *runner.DefaultSettings
+	ec := runner.NewEvalContext(settings, tmpDir)
+
+	if cache {
+		// sqlite file is present on disk
+		if info, err := os.Stat(getCachedDBPath(project.Id)); err != nil || info.Size() == 0 {
+			file, err := openTruncate(getCachedDBPath(project.Id))
+			if err != nil {
+				return err
+			}
+			file.Close()
```

Comment:
Why are you opening it just to close it?

Reply:
I am creating the sqlite file.

Reply:
Why does that need to happen?